azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingViewModel.kt

```diff
@@ -51,7 +55,19 @@ internal class CallingViewModel(
         floatingHeaderViewModel.switchFloatingHeader()
     }
 
+    fun requestCallEndOnBackPressed() {
+        confirmLeaveOverlayViewModel.requestExitConfirmation()
+    }
+
     fun requestCallEnd() {
```

Comment:
Please make sure this is called only on leave button click, not on back navigation.

Reply:
Added change to always show confirmation on navigation back click.